### PR TITLE
[Merged by Bors] - Fix: added a sleep to wait for OPA to fetch the bundle

### PR DESCRIPTION
**Pull request by fhennig**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by fhennig** [comment]
shellcheck raises a warning: https://www.shellcheck.net/wiki/SC2064

But the way it works is actually correct in this case.

File: docs/modules/getting_started/examples/code/getting_started.sh.j2 [hunk 1]
```diff
@@ -52,9 +52,14 @@ echo "Applying the rule file"
 kubectl apply -f simple-rule.yaml
 # end::apply-rule-file[]
 
+# The bundle builder will update the bundle almost immediately, but OPA can take up to
+# max_delay_seconds: 20 (see ConfigMap)
+# to poll the bundle
+sleep 21
```
Comment (maltesander) [inline docs/modules/getting_started/examples/code/getting_started.sh.j2:58]: https://github.com/stackabletech/opa-operator/blob/c49c5ae89b71990077364962a86d7f887eb0823f/rust/operator-binary/src/controller.rs#L596

Currently the OPA is configured to randomly poll every 10 to 20 seconds. Worst case you can have ~40 seconds.

**Review by maltesander — CHANGES_REQUESTED** [review]

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**